seacrowd/sea_datasets/iapp_squad/iapp_squad.py

```diff
@@ -0,0 +1,141 @@
+# coding=utf-8
+import json
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_DATASETNAME = "iapp_squad"
+_CITATION = """\
+            @dataset
+            {
+              kobkrit_viriyayudhakorn_2021_4539916,
+              author       = {Kobkrit Viriyayudhakorn and Charin Polpanumas},
+              title        = {iapp_wiki_qa_squad},
+              month        = feb,
+              year         = 2021,
+              publisher    = {Zenodo},
+              version      = 1,
+              doi          = {10.5281/zenodo.4539916},
+              url          = {https://doi.org/10.5281/zenodo.4539916}
+            }
```

Comment:
Small nit: can you also remove the leading whitespace here? I think it wouldn't render well when printed. Same with the `_DESCRIPTION` string